### PR TITLE
Add encryption feature

### PR DESCRIPTION
**Pull request by tkiapril**
For those who use password managers without functionality to execute custom commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +44,11 @@ You can use the following format to pass login parameters where applicable.
 * clipboard,text
 * clipboard,json
 * interactive
+* enc,(text|json),(path or - for stdin)[,passphrase]
+
+For the --enc parameter, you can use the following format.
+* path[,password]
+* user[+p[+..]],(text|json),(path or - for stdout)[,passphrase][;user[+p[+..],..]]
```
Comment (Soreepeong) [inline README.md:51]: Need some examples, including descriptions on what would each example do.

File: xivlogin.py [hunk 2]
```diff
@@ -29,6 +29,11 @@
 * clipboard,text
 * clipboard,json
 * interactive
+* enc,(text|json),(path or - for stdin)[,passphrase]
+
+For the --enc parameter, you can use the following format.
+* path[,password]
+* user[+p[+..]],(text|json),(path or - for stdout)[,passphrase][;user[+p[+..],..]]
```
Comment (Soreepeong) [inline xivlogin.py:39]: ^

File: README.md (hunk 1 above)
Comment (Soreepeong) [inline README.md:51]: Need some examples, including descriptions on what would each example do.

**Review by Soreepeong — CHANGES_REQUESTED** [review]

File: xivlogin.py (hunk 2 above)
Comment (tkiapril) [reply xivlogin.py:36]: Done!

File: README.md (hunk 1 above)
Comment (tkiapril) [reply README.md:51]: Done!

**Pull request merged**